packages/app/src/app/pages/Dashboard/Content/routes/Settings/TeamSettings/RegistryPreset/Custom.tsx

```diff
@@ -0,0 +1,85 @@
+import * as React from 'react';
+import { Select, Input, Text } from '@codesandbox/components';
+import { AuthType, RegistryType } from 'app/graphql/types';
+import { RegistryPresetProps } from '.';
+import { CustomFormField } from '../RegistryForm';
+
+export const CustomRegistryPreset = ({
+  registryUrl,
+  setRegistryUrl,
+  disabled,
+  authenticationType,
+  setAuthenticationType,
+  authKey,
+  setAuthKey,
+  setRegistryType,
+}: RegistryPresetProps) => {
+  React.useEffect(() => {
+    setRegistryType(RegistryType.Custom);
+  }, [setRegistryType]);
+  return (
+    <>
+      <div>
```

Comment:
Yah! Will do!

Reply:
Hmm, it made the bottom text show 2px closer to the input field (`8px` instead of `10px`), the `10px` felt like it breathes more, so I went with `div`